package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "cro-nodelib",
+  "version": "0.1.0",
+  "description": "Crypto.com Chain library for Node.js",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/crypto-com/cro-nodelib"
+  },
+  "author": "Calvin Lau <calvin@crypto.com>",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "ow": "^0.15.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.5",
```

Comment:
no need for fixed version dependencies yet?

Reply:
I will pin the dependencies in project setup.